float64[] lidar_data
